cmd/lncli/commands.go

```diff
@@ -2643,7 +2643,7 @@ func queryRoutes(ctx *cli.Context) error {
 		return fmt.Errorf("amt argument missing")
 	}
 
-	feeLimit, err := retrieveFeeLimit(ctx)
+	feeLimit, err := retrieveFeeLimitLegacy(ctx)
```

Comment:
do we keep this around bc there is no send to route on routerrpc?

Reply:
It is around for `QueryRoutes` for which there is no version on `routerrpc` yet.

Reply:
maybe something to consider for 0.11, people still find the plural name confusing anyway

Reply:
yes a reminiscence of k-shortest...